src/node_sea.cc

```diff
@@ -78,6 +95,172 @@ std::tuple<int, char**> FixupArgsForSEA(int argc, char** argv) {
   return {argc, argv};
 }
 
+class PrinterTryCatch : public TryCatch {
+ public:
+  explicit PrinterTryCatch(Isolate* isolate)
+      : TryCatch(isolate), isolate_(isolate) {}
+  ~PrinterTryCatch() {
+    if (!HasCaught()) {
+      return;
+    }
+    PrintCaughtException(isolate_, isolate_->GetCurrentContext(), *this);
+  }
+
+ private:
+  Isolate* isolate_;
+};
+
+template <typename T, void (*function)(T*)>
+class PointerWithDeleter {
+ public:
+  explicit PointerWithDeleter(T* ptr) : ptr_(ptr) {}
+  T* get() const { return ptr_; }
+  ~PointerWithDeleter() { function(ptr_); }
+
+ private:
+  T* ptr_;
+};
+
+// We might want a lighter-weight JSON parser because we only looking into
```

Comment:
Would it make sense to add something like `simdutf` in the future? (Not only for this but for other functions as well)

Reply:
I can see that might be useful for package.json reading, but for SEA config reading, which is not on any performance-critical path, that's probably an overkill.